profile/README.md

```diff
@@ -0,0 +1,63 @@
+# Lombiq Technologies 
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core CMS. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
+
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips. You can still update your software, but it will be a whole lot faster and more power-efficient.
```

Comment:
I think we should specify "FPGA" for the computer chips as it's an important keyword.

Reply:
Do you mean to add as follows: ... into computer chips (FPGA)... or to add a small introduction with the expression behind  FPGA?

Reply:
The first option is enough.